_posts/2017-06-23-announcing-angular-master-class-in-houston.md

```diff
@@ -12,11 +12,6 @@ tags:
   - announcements
 author: christoph_burgdorf
 related_posts:
-  - Announcing Angular Master Class in Berlin
-  - Join our free meetup in Denmark
-  - Announcing Angular Master Class in Denmark
-  - Dominic joins thoughtram
-  - Announcing Angular Master Class in Freiburg
```

Comment:
As mentioned on Slack, those are auto-generated by tags. We don't maintain them manually. So as soon as we deploy these changes, those will be re-added. I also don't think it's too much of a problem. As the UI states that those are related articles.